pkg/util/provider/machinecontroller/machine_util.go

```diff
@@ -1508,6 +1577,34 @@ func (c *controller) tryMarkingMachineFailed(ctx context.Context, machine, clone
 	return machineutils.ShortRetry, err
 }
 
+func getLiveVolumeAttachmentsForNode(volAttachLister storagelisters.VolumeAttachmentLister, nodeName string) ([]*storagev1.VolumeAttachment, error) {
+	volAttachments, err := volAttachLister.List(labels.NewSelector())
+	if err != nil {
+		return nil, fmt.Errorf("cant list volume attachments for node %q: %w", nodeName, err)
```

Comment:
also log machine name pls
```suggestion
		return nil, fmt.Errorf("can't list volume attachments for node %q: %w", nodeName, err)
```

Reply:
Changed to

```go

func getLiveVolumeAttachmentsForNode(volAttachLister storagelisters.VolumeAttachmentLister, nodeName string, machineName string) ([]*storagev1.VolumeAttachment, error) {
	volAttachments, err := volAttachLister.List(labels.NewSelector())
	if err != nil {
		return nil, fmt.Errorf("cant list volume attachments for node %q, machine %q: %w", nodeName, machineName, err)
	}
	//...
}
```